float64 time
float64 x_global
float64 y_global
float64 yaw_global
float64 h
float64 vl
float64 u
float64 y
float64 v
float64 phi
float64 r
float64 rd